.gitignore

```diff
@@ -16,4 +16,9 @@ bin/
 obj/
 packages/
 
-*.DotSettings.user
\ No newline at end of file
+*.DotSettings.user
+
+code_generator/End2End/generated_schema/
+code_generator/End2End/GeneratedDependencies/Generated/
+code_generator/End2End/GeneratedDependencies/Generated_Schema/
```

Comment:
Rename this folder to `Generated_Csharp_Worker` or similar

Reply:
Done